app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.__hummer_level = hummer_level
+
+    def player_info(self) -> str:
+        return f"Dwarf warrior {self.nickname}. {self.nickname} has " \
+               f"a hummer of the {self.__hummer_level} level"
```

Comment:
Preferred to use parenthesis instead of \
```suggestion
        return (
            f"Dwarf warrior {self.nickname}. {self.nickname} has "
            f"a hummer of the {self.__hummer_level} level"
        )   
```
Fix in all places